server/server.go

```diff
@@ -972,25 +972,28 @@ func (s *Server) ActivePeers() (peers []string) {
 
 // isTrustedIssuer will check that the issuer is a trusted public key.
 // This is used to make sure an account was signed by a trusted operator.
-func (s *Server) isTrustedIssuer(issuer string) bool {
```

Comment:
You could add a little comment above that says that this function also returns "true" is bearer token are not allowed.

Reply:
added comment